backend/main/server/resources/Video.py

```diff
@@ -2,11 +2,28 @@
 from flask import request
 from main.server import db, cache, app
 from main.server.models import Video, VideoSchema
-from flask_jwt import jwt_required
 
 video_schema = VideoSchema()
 videos_schema = VideoSchema(many=True)
 
+def insertVideo(col, data):
```

Comment:
I'm not actually convinced that these insertion functions should be part of the individual resources. They pretty much directly touch the db and don't need to be launched with the backend code. I think it might be better to move them directly into `update_db.py`

Reply:
hmm, ok. I'll move these functions to their own place. Probably will make it easier that way too.

Reply:
I added them to main/utils/insert.py instead. Does that work?